src/nft/components/explore/Banner.tsx

```diff
@@ -33,17 +33,17 @@ const HeaderContainer = styled.div`
   display: flex;
   max-width: 500px;
   font-weight: 500;
-  font-size: 60px;
+  font-size: 72px;
   line-height: 73px;
   justify-content: start;
   align-items: start;
   padding-top: 40px;
   flex-shrink: 0;
   background: linear-gradient(180deg, rgba(255, 255, 255, 0.4) 0%, rgba(255, 255, 255, 0) 100%), #fc72ff;
   -webkit-background-clip: text;
-  -webkit-text-fill-color: transparent;
+  -webkit-text-fill-color: ${({ theme }) => theme.textPrimary};
```

Comment:
we shouldn't need this because we have auto prefixer

Reply:
actually without this, it doesn't work on chrome

Reply:
let me see why, that's odd to me